### PR TITLE
Known issues 2020-01-09

### PR DESCRIPTION
**Pull request by Simran-B**
- CentOS issues resolved since 3.4.8
- javascript.startup-directory issues in Windows client packages will be resolved in 3.4.10, 3.5.5, 3.6.1
- hypergraph fix in 3.6.1
- resetDeadJobs / Foxxmaster failover will be resolved in 3.4.10, 3.5.5, 3.6.1
- sync some of the copied files
- inclusion of arangoinspect in Windows client package
  https://github.com/arangodb/arangodb/pull/10835

### CODE REVIEW

File: 3.7/release-notes-known-issues36.md [hunk 1]
```diff
@@ -54,7 +55,7 @@ Other
...
+| **Date Added:** 2019-12-10 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The DMG package for macOS is not notarized, which prevents the execution of _ArangoDB3-CLI.app_ under macOS 10.15 (Catalina) with error message: "_ArangoDB3-CLI_ can't be opened because Apple cannot check it for malicious software" <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10561](https://github.com/arangodb/arangodb/issues/10561){:target="_blank"} |
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** V8-based binaries of the client packages (arangosh, arangoinspect, foxx-manager) have an incorrect default value for `javascript.startup-directory` and will thus not find the required JavaScript folder unless specified by the user. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.4.10, 3.5.5, 3.6.2 <br> **Reference:** [arangodb/release-qa#183](https://github.com/arangodb/release-qa/issues/183){:target="_blank"} (internal) |
```
Comment (jsteemann) [inline 3.7/release-notes-known-issues36.md]: I think it should be 3.4.9, 3.5.4 and 3.6.1 here?

Comment (Simran-B) [reply 3.7/release-notes-known-issues36.md]: 3.4.10, 3.5.5, 3.6.1 as per Vadim (changes too close to 3.4/3.5 release, will not be included)

File: 3.4/release-notes-known-issues34.md [hunk 2]
```diff
@@ -60,6 +60,6 @@ Other
...
-| **Date Added:** 2020-01-07 <br> **Component:** Foxx <br> **Deployment Mode:** Cluster <br> **Description:** In case of a Foxxmaster failover, jobs in state `'progress'` are not reset to `'pending'` to restart execution. <br> **Affected Versions:** 3.4.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10800](https://github.com/arangodb/arangodb/pull/10800){:target="_blank"} |
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** V8-based binaries of the client packages (arangosh, arangoinspect, foxx-manager) have an incorrect default value for `javascript.startup-directory` and will thus not find the required JavaScript folder unless specified by the user. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.4.10, 3.5.5, 3.6.1 <br> **Reference:** [arangodb/release-qa#183](https://github.com/arangodb/release-qa/issues/183){:target="_blank"} (internal) |
```
Comment (KVS85) [inline 3.4/release-notes-known-issues34.md]: Fixed in 3.4.9, 3.5.4, 3.6.1.

File: 3.4/release-notes-known-issues34.md [hunk 3]
```diff
@@ -60,6 +60,6 @@ Other
...
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The client packages for Windows miss the arangoinspect binary. As a workaround, you can run arangosh with the following options:<br>`arangosh --server.authentication false --server.ask-jwt-secret --javascript.client-module inspector.js …` <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.3.25, 3.4.9, 3.5.4, 3.6.1 <br> **Reference:** [arangodb/arangodb#10835](https://github.com/arangodb/arangodb/pull/10835){:target="_blank"} |
```
Comment (KVS85) [inline 3.4/release-notes-known-issues34.md]: Fixed in 3.3.25, 3.4.10, 3.5.5, 3.6.1.

File: 3.5/release-notes-known-issues34.md [hunk 4]
```diff
@@ -60,6 +60,6 @@ Other
...
-| **Date Added:** 2020-01-07 <br> **Component:** Foxx <br> **Deployment Mode:** Cluster <br> **Description:** In case of a Foxxmaster failover, jobs in state `'progress'` are not reset to `'pending'` to restart execution. <br> **Affected Versions:** 3.4.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10800](https://github.com/arangodb/arangodb/pull/10800){:target="_blank"} |
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** V8-based binaries of the client packages (arangosh, arangoinspect, foxx-manager) have an incorrect default value for `javascript.startup-directory` and will thus not find the required JavaScript folder unless specified by the user. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.4.10, 3.5.5, 3.6.1 <br> **Reference:** [arangodb/release-qa#183](https://github.com/arangodb/release-qa/issues/183){:target="_blank"} (internal) |
```
Comment (KVS85) [inline 3.5/release-notes-known-issues34.md]: Fixed in 3.4.9, 3.5.4, 3.6.1.

File: 3.5/release-notes-known-issues34.md [hunk 5]
```diff
@@ -60,6 +60,6 @@ Other
...
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The client packages for Windows miss the arangoinspect binary. As a workaround, you can run arangosh with the following options:<br>`arangosh --server.authentication false --server.ask-jwt-secret --javascript.client-module inspector.js …` <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.3.25, 3.4.9, 3.5.4, 3.6.1 <br> **Reference:** [arangodb/arangodb#10835](https://github.com/arangodb/arangodb/pull/10835){:target="_blank"} |
```
Comment (KVS85) [inline 3.5/release-notes-known-issues34.md]: Fixed in 3.3.25, 3.4.10, 3.5.5, 3.6.1.

File: 3.5/release-notes-known-issues35.md [hunk 6]
```diff
@@ -72,6 +71,6 @@ Other
...
-| **Date Added:** 2020-01-07 <br> **Component:** Foxx <br> **Deployment Mode:** Cluster <br> **Description:** In case of a Foxxmaster failover, jobs in state `'progress'` are not reset to `'pending'` to restart execution. <br> **Affected Versions:** 3.4.x, 3.5.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10800](https://github.com/arangodb/arangodb/pull/10800){:target="_blank"} |
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** V8-based binaries of the client packages (arangosh, arangoinspect, foxx-manager) have an incorrect default value for `javascript.startup-directory` and will thus not find the required JavaScript folder unless specified by the user. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.4.10, 3.5.5, 3.6.1 <br> **Reference:** [arangodb/release-qa#183](https://github.com/arangodb/release-qa/issues/183){:target="_blank"} (internal) |
```
Comment (KVS85) [inline 3.5/release-notes-known-issues35.md]: Fixed in 3.4.9, 3.5.4, 3.6.1.

File: 3.5/release-notes-known-issues35.md [hunk 7]
```diff
@@ -72,6 +71,6 @@ Other
...
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The client packages for Windows miss the arangoinspect binary. As a workaround, you can run arangosh with the following options:<br>`arangosh --server.authentication false --server.ask-jwt-secret --javascript.client-module inspector.js …` <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.3.25, 3.4.9, 3.5.4, 3.6.1 <br> **Reference:** [arangodb/arangodb#10835](https://github.com/arangodb/arangodb/pull/10835){:target="_blank"} |
```
Comment (KVS85) [inline 3.5/release-notes-known-issues35.md]: Fixed in 3.3.25, 3.4.10, 3.5.5, 3.6.1.

File: 3.6/release-notes-known-issues34.md [hunk 8]
```diff
@@ -60,6 +60,6 @@ Other
...
-| **Date Added:** 2020-01-07 <br> **Component:** Foxx <br> **Deployment Mode:** Cluster <br> **Description:** In case of a Foxxmaster failover, jobs in state `'progress'` are not reset to `'pending'` to restart execution. <br> **Affected Versions:** 3.4.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10800](https://github.com/arangodb/arangodb/pull/10800){:target="_blank"} |
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** V8-based binaries of the client packages (arangosh, arangoinspect, foxx-manager) have an incorrect default value for `javascript.startup-directory` and will thus not find the required JavaScript folder unless specified by the user. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.4.10, 3.5.5, 3.6.1 <br> **Reference:** [arangodb/release-qa#183](https://github.com/arangodb/release-qa/issues/183){:target="_blank"} (internal) |
```
Comment (KVS85) [inline 3.6/release-notes-known-issues34.md]: Fixed in 3.4.9, 3.5.4, 3.6.1.

File: 3.6/release-notes-known-issues34.md [hunk 9]
```diff
@@ -60,6 +60,6 @@ Other
...
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The client packages for Windows miss the arangoinspect binary. As a workaround, you can run arangosh with the following options:<br>`arangosh --server.authentication false --server.ask-jwt-secret --javascript.client-module inspector.js …` <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.3.25, 3.4.9, 3.5.4, 3.6.1 <br> **Reference:** [arangodb/arangodb#10835](https://github.com/arangodb/arangodb/pull/10835){:target="_blank"} |
```
Comment (KVS85) [inline 3.6/release-notes-known-issues34.md]: Fixed in 3.3.25, 3.4.10, 3.5.5, 3.6.1.

File: 3.6/release-notes-known-issues35.md [hunk 10]
```diff
@@ -70,7 +69,8 @@ Other
...
-| **Date Added:** 2020-01-07 <br> **Component:** Foxx <br> **Deployment Mode:** Cluster <br> **Description:** In case of a Foxxmaster failover, jobs in state `'progress'` are not reset to `'pending'` to restart execution. <br> **Affected Versions:** 3.4.x, 3.5.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10800](https://github.com/arangodb/arangodb/pull/10800){:target="_blank"} |
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** V8-based binaries of the client packages (arangosh, arangoinspect, foxx-manager) have an incorrect default value for `javascript.startup-directory` and will thus not find the required JavaScript folder unless specified by the user. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.4.10, 3.5.5, 3.6.1 <br> **Reference:** [arangodb/release-qa#183](https://github.com/arangodb/release-qa/issues/183){:target="_blank"} (internal) |
```
Comment (KVS85) [inline 3.6/release-notes-known-issues35.md]: Fixed in 3.4.9, 3.5.4, 3.6.1.

File: 3.6/release-notes-known-issues35.md [hunk 11]
```diff
@@ -70,7 +69,8 @@ Other
...
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The client packages for Windows miss the arangoinspect binary. As a workaround, you can run arangosh with the following options:<br>`arangosh --server.authentication false --server.ask-jwt-secret --javascript.client-module inspector.js …` <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.3.25, 3.4.9, 3.5.4, 3.6.1 <br> **Reference:** [arangodb/arangodb#10835](https://github.com/arangodb/arangodb/pull/10835){:target="_blank"} |
```
Comment (KVS85) [inline 3.6/release-notes-known-issues35.md]: Fixed in 3.3.25, 3.4.10, 3.5.5, 3.6.1.

File: 3.6/release-notes-known-issues36.md [hunk 12]
```diff
@@ -57,6 +56,6 @@ Other
...
-| **Date Added:** 2020-01-07 <br> **Component:** Foxx <br> **Deployment Mode:** Cluster <br> **Description:** In case of a Foxxmaster failover, jobs in state `'progress'` are not reset to `'pending'` to restart execution. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10800](https://github.com/arangodb/arangodb/pull/10800){:target="_blank"} |
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** V8-based binaries of the client packages (arangosh, arangoinspect, foxx-manager) have an incorrect default value for `javascript.startup-directory` and will thus not find the required JavaScript folder unless specified by the user. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.4.10, 3.5.5, 3.6.1 <br> **Reference:** [arangodb/release-qa#183](https://github.com/arangodb/release-qa/issues/183){:target="_blank"} (internal) |
```
Comment (KVS85) [inline 3.6/release-notes-known-issues36.md]: Fixed in 3.4.9, 3.5.4, 3.6.1.

File: 3.6/release-notes-known-issues36.md [hunk 13]
```diff
@@ -57,6 +56,6 @@ Other
...
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The client packages for Windows miss the arangoinspect binary. As a workaround, you can run arangosh with the following options:<br>`arangosh --server.authentication false --server.ask-jwt-secret --javascript.client-module inspector.js …` <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.3.25, 3.4.9, 3.5.4, 3.6.1 <br> **Reference:** [arangodb/arangodb#10835](https://github.com/arangodb/arangodb/pull/10835){:target="_blank"} |
```
Comment (KVS85) [inline 3.6/release-notes-known-issues36.md]: Fixed in 3.3.25, 3.4.10, 3.5.5, 3.6.1.

File: 3.7/release-notes-known-issues34.md [hunk 14]
```diff
@@ -57,3 +59,7 @@ Other
...
+| **Date Added:** 2019-12-10 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The DMG package for macOS is not notarized, which prevents the execution of _ArangoDB3-CLI.app_ under macOS 10.15 (Catalina) with error message: "_ArangoDB3-CLI_ can't be opened because Apple cannot check it for malicious software" <br> **Affected Versions:** 3.3.x, 3.4.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10561](https://github.com/arangodb/arangodb/issues/10561){:target="_blank"} |
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** V8-based binaries of the client packages (arangosh, arangoinspect, foxx-manager) have an incorrect default value for `javascript.startup-directory` and will thus not find the required JavaScript folder unless specified by the user. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.4.10, 3.5.5, 3.6.1 <br> **Reference:** [arangodb/release-qa#183](https://github.com/arangodb/release-qa/issues/183){:target="_blank"} (internal) |
```
Comment (KVS85) [inline 3.7/release-notes-known-issues34.md]: Fixed in 3.4.9, 3.5.4, 3.6.1.

File: 3.7/release-notes-known-issues34.md [hunk 15]
```diff
@@ -57,3 +59,7 @@ Other
...
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The client packages for Windows miss the arangoinspect binary. As a workaround, you can run arangosh with the following options:<br>`arangosh --server.authentication false --server.ask-jwt-secret --javascript.client-module inspector.js …` <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.3.25, 3.4.9, 3.5.4, 3.6.1 <br> **Reference:** [arangodb/arangodb#10835](https://github.com/arangodb/arangodb/pull/10835){:target="_blank"} |
```
Comment (KVS85) [inline 3.7/release-notes-known-issues34.md]: Fixed in 3.3.25, 3.4.10, 3.5.5, 3.6.1.

File: 3.7/release-notes-known-issues35.md [hunk 16]
```diff
@@ -69,4 +69,8 @@ Other
...
+| **Date Added:** 2019-12-10 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The DMG package for macOS is not notarized, which prevents the execution of _ArangoDB3-CLI.app_ under macOS 10.15 (Catalina) with error message: "_ArangoDB3-CLI_ can't be opened because Apple cannot check it for malicious software" <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10561](https://github.com/arangodb/arangodb/issues/10561){:target="_blank"} |
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** V8-based binaries of the client packages (arangosh, arangoinspect, foxx-manager) have an incorrect default value for `javascript.startup-directory` and will thus not find the required JavaScript folder unless specified by the user. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.4.10, 3.5.5, 3.6.1 <br> **Reference:** [arangodb/release-qa#183](https://github.com/arangodb/release-qa/issues/183){:target="_blank"} (internal) |
```
Comment (KVS85) [inline 3.7/release-notes-known-issues35.md]: Fixed in 3.4.9, 3.5.4, 3.6.1.

File: 3.7/release-notes-known-issues35.md [hunk 17]
```diff
@@ -69,4 +69,8 @@ Other
...
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The client packages for Windows miss the arangoinspect binary. As a workaround, you can run arangosh with the following options:<br>`arangosh --server.authentication false --server.ask-jwt-secret --javascript.client-module inspector.js …` <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.3.25, 3.4.9, 3.5.4, 3.6.1 <br> **Reference:** [arangodb/arangodb#10835](https://github.com/arangodb/arangodb/pull/10835){:target="_blank"} |
```
Comment (KVS85) [inline 3.7/release-notes-known-issues35.md]: Fixed in 3.3.25, 3.4.10, 3.5.5, 3.6.1.

File: 3.7/release-notes-known-issues36.md [hunk 18]
```diff
@@ -54,7 +55,7 @@ Other
...
+| **Date Added:** 2019-12-10 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The DMG package for macOS is not notarized, which prevents the execution of _ArangoDB3-CLI.app_ under macOS 10.15 (Catalina) with error message: "_ArangoDB3-CLI_ can't be opened because Apple cannot check it for malicious software" <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10561](https://github.com/arangodb/arangodb/issues/10561){:target="_blank"} |
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** V8-based binaries of the client packages (arangosh, arangoinspect, foxx-manager) have an incorrect default value for `javascript.startup-directory` and will thus not find the required JavaScript folder unless specified by the user. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.4.10, 3.5.5, 3.6.1 <br> **Reference:** [arangodb/release-qa#183](https://github.com/arangodb/release-qa/issues/183){:target="_blank"} (internal) |
```
Comment (KVS85) [inline 3.7/release-notes-known-issues36.md]: Fixed in 3.4.9, 3.5.4, 3.6.1.

File: 3.7/release-notes-known-issues36.md [hunk 19]
```diff
@@ -54,7 +55,7 @@ Other
...
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The client packages for Windows miss the arangoinspect binary. As a workaround, you can run arangosh with the following options:<br>`arangosh --server.authentication false --server.ask-jwt-secret --javascript.client-module inspector.js …` <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.3.25, 3.4.9, 3.5.4, 3.6.1 <br> **Reference:** [arangodb/arangodb#10835](https://github.com/arangodb/arangodb/pull/10835){:target="_blank"} |
```
Comment (KVS85) [inline 3.7/release-notes-known-issues36.md]: Fixed in 3.3.25, 3.4.10, 3.5.5, 3.6.1.

**Pull request merged**